src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -2155,6 +2155,34 @@ public static void DCS_CircularTypes_PreserveObjectReferences_False()
         Assert.True(root2.Children[0] == root2.Children[2], "root2.Children[0] != root2.Children[2]");
     }
 
+    [Fact]
```

Comment:
Same as for DCJS, can you create a CultureInfo object on the full framework, access each of it's public properties to make sure anything lazily created is created, then serialize it. Take the output of that serialization and make sure it can be deserialized in a test here.